src/bridge/cockpitpeer.c

```diff
@@ -454,6 +455,7 @@ on_transport_control (CockpitTransport *transport,
   const gchar *cookie = NULL;
   gboolean forward = FALSE;
   gboolean handled = FALSE;
+  gboolean privileged = FALSE;
```

Comment:
Nitpick: This could move into the `if` block, we don't need it anywhere outside.

Reply:
We don't seem to be doing that in other places, so I'd leave it here.